ansible/roles/uan_hardening/tasks/disable_ssh_out_uan_to_nmn_lb.yml

```diff
@@ -0,0 +1,58 @@
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+---
+- hosts: Application_UAN
+  gather_facts: no
+  tasks:
+    - name: "Get NMNLB subnet info from SLS"
+      local_action:
+        module: uri
+        url: "http://cray-sls/v1/search/networks?name=NMNLB"
+        method: GET
+      register: sls_nmnlb
+      failed_when: sls_nmnlb.status != 200
+      run_once: true
+
+    - name: "Set NMNLB subnet details"
+      set_fact:
+        nmnlb_subnet_details: "{{ item.ExtraProperties.Subnets }}"
+      loop: "{{ sls_nmnlb.json }}"
+
+    - name: "Set NMNLB subnet facts from SLS API retrieved details"
+      set_fact:
+        nmnlb_network_cidr: "{{ item.CIDR }}"
+      loop: "{{ nmnlb_subnet_details }}"
```

Comment:
To be safe, should probably have a when clause here.   Right night now it works because there is only one subnet in the NMNLB network.   But if we ever decide to break it up into multiple metallb pools, for example, then it will be non-deterministic.

Something like:
`when:  item.FullName == "NMN MetalLB"`

Reply:
Completed adding and testing this. Tested succesfully